change/@fluentui-react-charting-3fdcb2d7-fde5-4ec3-bf0e-1e9d32cd1194.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Have you put this change behind the optimize flag?

Reply:
No , Conditional import are not possible with current ES version 5 . They are supported from Version 6 Onwards. https://mariusschulz.com/blog/dynamic-import-expressions-in-typescript